adr/ADR-20.md

```diff
@@ -388,13 +395,20 @@ Watch(opts ...WatchOpt) -> ObjectWatcher
 **List**
 
 List will list all the objects in this store.
- 
-* Should not include deleted objects.
+* Do not include deleted objects, except with optional convenience methods.
```

Comment:
When listing objects, filter those objects that have been deleted. If necessary or requested, provide convenience methods or optional arguments to include them if the API user desires them.

Reply:
fixed